docs/source/guides/staking_guide.rst

```diff
@@ -298,52 +332,72 @@ allow **re-staking** to be disabled until the release period begins, even if you
 No action is needed to release the re-staking lock once the release period begins.
 
 
-Modifying Active Stakes
-~~~~~~~~~~~~~~~~~~~~~~~~
+Prolong
+*******
 
-Several administrative operations can be performed on active stakes:
+Existing stakes can be extended by a number of periods as long as the resulting
+stake's duration is not longer than the maximum. To prolong an existing stake's duration:
 
-+----------------------+-------------------------------------------------------------------------------+
-| Action               |  Description                                                                  |
-+======================+===============================================================================+
-|  ``divide``          | Create a new stake from part of an existing one                               |
-+----------------------+-------------------------------------------------------------------------------+
-|  ``prolong``         | Prolong an existing stake's duration                                          |
-+----------------------+-------------------------------------------------------------------------------+
-|  ``winddown``        | Manage winding down of stakes                                                 |
-+----------------------+-------------------------------------------------------------------------------+
+.. code:: bash
 
+    (nucypher)$ nucypher stake prolong
 
-Divide
-******
 
-To divide an existing stake into discrete substakes:
+Wind Down
+**********
+
+Wind down is *diabled* by default. To start winding down an existing stake:
 
 .. code:: bash
 
-    (nucypher)$ nucypher stake divide
+    (nucypher)$ nucypher stake winddown
 
 
-Prolong
-*******
+Divide
+******
 
-To prolong an existing stake's duration:
+Existing stakes can be divided into smaller sub-stakes, with added value and duration. To divide an existing stake:
 
 .. code:: bash
 
-    (nucypher)$ nucypher stake prolong
+    (nucypher)$ nucypher stake divide --hw-wallet
```

Comment:
Previous commands didn't use the --hw-wallet flag

Reply:
:heavy_check_mark: